website/assets/styles/pages/pricing.less

```diff
@@ -35,29 +31,34 @@
     color: #192147;
     font-weight: bold;
     user-select: none;
-    transition: 0.2s linear;
-    -o-transition: 0.2s linear;
-    -ms-transition: 0.2s linear;
-    -moz-transition: 0.2s linear;
-    -webkit-transition: 0.2s linear;
+    transition: 0.2s ease-in-out;
+    -o-transition: 0.2s ease-in-out;
+    -ms-transition: 0.2s ease-in-out;
+    -moz-transition: 0.2s ease-in-out;
+    -webkit-transition: 0.2s ease-in-out;
   }
 
   #arrow:before {
-    content: url('/images/arrow-right-16x16@2x.png');
+    content: url("/images/arrow-right-16x16@2x.png");
     transform: scale(0.5);
     position: absolute;
     top: 12px;
     left: 80%; // <--- here
+    transition: 0.2s ease-in-out;
+    -o-transition: 0.2s ease-in-out;
+    -ms-transition: 0.2s ease-in-out;
+    -moz-transition: 0.2s ease-in-out;
+    -webkit-transition: 0.2s ease-in-out;
     /*   opacity: 0; */
   }
 
   #arrow:hover:before {
     left: 82%; // <--- here
-    transition: 0.2s linear;
-    -o-transition: 0.2s linear;
-    -ms-transition: 0.2s linear;
-    -moz-transition: 0.2s linear;
-    -webkit-transition: 0.2s linear;
+    transition: 0.2s ease-in-out;
+    -o-transition: 0.2s ease-in-out;
+    -ms-transition: 0.2s ease-in-out;
+    -moz-transition: 0.2s ease-in-out;
+    -webkit-transition: 0.2s ease-in-out;
```

Comment:
@gillespi314 @RachelElysia btw y'all I think there's a bundled mixin for doing these, so you don't have to expand all the vendor prefixes  (not a blocker to merging this, just fyi)